packaging/windows/buildpackage.sh

```diff
@@ -17,9 +17,9 @@ TAG="$1"
 OUTPUTDIR="$2"
 SRCDIR="$(pwd)/../.."
 BUILTDIR="$(pwd)/build"
+LAUNCHERBUILTDIR="$(pwd)/launcher"
 ARTWORK_DIR="$(pwd)/../artwork/"
 
-LAUNCHER_LIBS="-r:System.dll -r:System.Drawing.dll -r:System.Windows.Forms.dll -r:${BUILTDIR}/OpenRA.Game.dll"
```

Comment:
This will be covered by #18865, so expect to need to rebase and resolve a few more conflicts once that has been merged.  It would also be good to wait for #18861 before rebasing, so you can then add the runtime detection there.